package.json

```diff
@@ -13,7 +13,8 @@
     "dist-all": "build -mwl --x64 --ia32",
     "pack": "build --dir",
     "dist": "build --x64 --ia32",
-    "test": "mocha test"
+    "test": "eslint . && mocha test",
```

Comment:
I switched to Windows and tested it with node v6.9.4 and npm 3.10.6 and it worked. Strange but let's remove it for now from test script since I guess istanbul can't process eslint result. It's still useful to have eslint locally though, then an editor can use it for error checking.